### PR TITLE
Ingest data even if only image CSVs are present

### LINKED ISSUE #89: Allow use of cytominer-database on just Image CSVs
It _looks_ like in [this function](https://github.com/cytomining/cytominer-database/blob/master/cytominer_database/utils.py#L94-L98) cytominer-database mandates that you have object CSVs, though I have not explicitly tested it.  

If it does require object CSVs, we should make that optional.  If it doesn't and I'm reading the code wrong, will happily close.

### PR DESCRIPTION
**Pull request by mcquin**
Resolves #89 

Allows cytominer_database to ingest only image CSVs. Introduces default configuration values for image and experiment CSV files. 

Additionally resolves a difference between documentation and implementation asserting that `config` should be a file (in implementation, it was a config instance). This simplifies both the code from the user perspective (no need for the user to read the config on their own) and the setting/handling of default configuration values.

### CODE REVIEW

**Review by shntnu — COMMENTED** [review]
Looks good! Once the test is in place, we're all set

**Comment by bethac07** [comment]
My only comments- 
1) The "Experiment" CSVs now seem to be included as a database table; I dont' care at all but did not know if that was expected.
2) I have to pass the `--no-munge` flag in order to get it to work; this is fine but should be documented.

**Comment by mcquin** [comment]
Good points. I'll address those issues first thing tomorrow.

As a temporary work-around, you can filter our the experiment CSVs by
adding an "experiment = Experiment.csv" line under the filenames section of
the config.

**Review by shntnu — COMMENTED** [review]
The code looks so much simpler, especially the tests!
Great idea creating a default config. I have a comment in there, but the rest looks great!

File: tests/conftest.py [hunk 1]
```diff
@@ -8,48 +8,90 @@ def pytest_addoption(parser):
...
+        "data_dir": "tests/data_c",
+        "image_csv": "Image.csv",
+        "ingest": [
+            {
+                "nrows": 8,
+                "ncols": 229,
+                "table": "Image"
+            }
+        ],
         "munge": True
```
Comment (shntnu) [inline tests/conftest.py:95]: Shouldn't this false `False`? @mcquin

Comment (mcquin) [reply tests/conftest.py:95]: Either work. This is meant to test that it doesn't vomit when munge is passed (True) but there aren't any object CSVs in the config.

Comment (shntnu) [reply tests/conftest.py:95]: Make sense. Lets leave it as is in which case

**Review by shntnu — APPROVED** [review]

**Pull request merged**